ot-node.js

```diff
@@ -83,12 +92,12 @@ class OTNode {
     async initializeDataModule() {
         try {
             const dataService = this.container.resolve('dataService');
-            if (!this.config.data) {
-                this.logger.warn('Data module not initialized, no implementation is provided');
-            }
+            // if (!this.config.data) {
```

Comment:
The idea here was to have optional modules that are not required

Reply:
i agree, but i think that ot-node should not be responsible for checking if nodule is required or not. Each module manager (in this case service) should be responsible to create instance of module if required in configuration. What do you think?